### PR TITLE
Use conda

### PR DESCRIPTION
**Pull request by sjanssen2**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +16,8 @@ install:
   - conda install -q nose
   - pip install -q https://github.com/qiime2/q2lint/archive/master.zip
   - pip install -q flake8
+  - conda install -c sjanssen2 -c bioconda fragment-insertion
```
Comment (sjanssen2) [inline .travis.yml]: need to change to biocore, once its merged and uploaded

File: ci/recipe/meta.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{% set version = '0.3.0' %}
```
Comment (sjanssen2) [inline ci/recipe/meta.yaml]: I think I should revert to own version numbers

**Comment by sjanssen2** [comment]
@antgonza could you review this one please? I want to make sure the same installation process via conda is now also used in this q2 plugin

**Comment by sjanssen2** [comment]
@josenavas @antgonza @tanaes I would greatly appreciate your reviews

File: ci/recipe/meta.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
-{% set qiime2release = '2017.10' %}
+{% set version = '0.2017.12' %}
```
Comment (antgonza) [inline ci/recipe/meta.yaml]: This is the plugin version, I'm OK with matching with q2  ...

**Pull request merged**